generate.py

```diff
@@ -1,18 +1,47 @@
-from Crypto.Cipher import AES
-from Crypto.Util.Padding import pad, unpad
-from pyotp import TOTP
-import base64
+#!/bin/env python3
+#
+# This code is based on article
+# https://huggable.tech/blog/extracting-fortitoken-mobile-totp-secret
```

Comment:
this article does not exist (anymore?)

Reply:
Yesterday it was in google cache and now it's gone.
Here is the link to web.archive.org:
https://web.archive.org/web/20221022175145/https://huggable.tech/blog/extracting-fortitoken-mobile-totp-secret

Reply:
Should I place link to web.archive.org instead then?